react/components/Table/BulkActions.js

```diff
@@ -1,17 +1,33 @@
 import React, { PureComponent } from 'react'
 import PropTypes from 'prop-types'
 import classNames from 'classnames'
+import Sticky from 'react-stickynode'
 
 import Button from '../Button'
 import ButtonWithIcon from '../ButtonWithIcon'
 import ActionMenu from '../ActionMenu'
 import Close from '../icon/Close'
 
 import BulkActionsSelectedRows from './BulkActionsSelectedRows'
+import { TABLE_CONTENT_CLASS } from './SimpleTable'
 
 const close = <Close />
 
 class BulkActions extends PureComponent {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      sticky: false,
```

Comment:
Try to change the name of the variable to something less generic.

Reply:
Done!